android/debugging_tips.md

```diff
@@ -0,0 +1,14 @@
+# Debugging Tips
+
+
+### Profiling code snippits
```

Comment:
Do you think we should move this to `android/android_guide.md` in the "Intermediate topics" section? e.g. I was going to add tips for reading the Android source code.

But it might be harder to find the debugging tips because they'd probably be overlooked.